README.md

```diff
@@ -337,6 +337,36 @@ interface {
 }
 ```
 
+#### [Begin Firmware Update](https://zwave-js.github.io/node-zwave-js/#/api/node?id=beginfirmwareupdate)
+
+##### Format must be guessed
+
+[compatible with schema version: 5+]
+
+```ts
+interface {
+  messageId: string;
+  command: "node.begin_firmware_update_guess_format";
+  nodeId: number;
+  firmwareFilename: string;
+  firmwareFile: string; // use base64 encoding for the file
+}
+```
+
+##### Format is known
+
+[compatible with schema version: 5+]
+
+```ts
+interface {
+  messageId: string;
+  command: "node.begin_firmware_update_known_format";
```

Comment:
that was my original plan, but you don't need the filename in the case of the file format, and I thought it would be too confusing to have two parameters that were exclusively optional. But I guess there's no harm in always collecting the filename since the caller should have it.